tests/Alchemy/Tests/Phrasea/Controller/MinifierTest.php

```diff
@@ -49,13 +49,21 @@ public function testFileMinificationWithoutParamsShouldReturnA400()
 
     public function testFileMinificationWithUnknownGroupShouldReturnA500()
     {
-        self::$DI['client']->request('GET', '/include/minify/?g=prout');
-        $this->assertEquals(500, self::$DI['client']->getResponse()->getStatusCode());
+        // there is an exception thrown
+        try {
+            $this->fail('An exception should have been raised');
```

Comment:
no. this will jump directly to the catch, so the request is not played.
The request doest NOT throw an exception, but dumps a bunch of " phraseanet.ERROR: Unable to generate data...". Thats is that what we don't want to see during phpunit.